\documentclass[11pt,twoside,openright,letterpaper]{memoir}
\setsecnumdepth{subsection} % include numbering on subsections
\setcounter{tocdepth}{3}

% Use 1/2-inch margins.
% \usepackage[margin=1in,lmargin=1.5in,showframe]{geometry} % ,showframe % for margin boundary box
\usepackage[margin=1in,lmargin=1.5in]{geometry} % ,showframe % for margin boundary box

\usepackage{titling}
\usepackage{helvet} % Use Helvetica font.
\renewcommand{\familydefault}{\sfdefault}
% \usepackage{setspace}
\usepackage{tabularx}
\usepackage{array} % get left justified columns
\newcolumntype{P}[1]{>{\raggedright\arraybackslash}p{#1}} % get left justified columns
\usepackage{pdfpages}
\raggedbottom % solve inconsistent paragraph spacing issue

%%% bibliography:
\usepackage[super,comma,sort&compress,sectionbib]{natbib}
\usepackage{chapterbib}
% switch bibliography numbering to 1. from [1] :
\makeatletter \renewcommand\@biblabel[1]{#1.} \makeatother
\setlength{\bibsep}{0pt}

% math:
\usepackage{amsmath} % for multline
\usepackage{mathtools}
\usepackage{siunitx} % for math formatting
\usepackage{newtxtext,newtxmath} % formats numbers in math mode to consistent font
\newcases{cdcases}{\quad}{$\hfil\displaystyle{##}$\hfil}{$\displaystyle{##}$\hfil}{\lbrace}{.} % for gene conv model

% graphics:
\usepackage{graphicx}
\usepackage{epstopdf} %converting to PDF
\usepackage[section]{placeins} % keep floats within sections
\usepackage{afterpage} % wrap around floats to have them on their own pages
\usepackage{wrapfig} % allow wrapfig and wraptable

% glossary:
\usepackage[toc,nonumberlist]{glossaries}
\makeglossaries

\usepackage{hyperref} % allow urls
\hypersetup{
    pdfauthor={CLCampbell},
%    pdfsubject={Some subject},
    pdftitle={},
%    pdfkeywords={LaTeX, PDF, hyperlinks}.
    citebordercolor = {1 1 1}, % no box around citations
    %,pdfborder = {0 0 1} % visible box around hyperlinks.
    %,pdfborderstyle={/S/U/W 1}
    colorlinks,
    citecolor=black,
    filecolor=black,
    linkcolor=black,
    urlcolor=black, % blue
    pdfpagemode=UseOutlines
}
% \usepackage{hypcap} % anchor links at image, not caption
\usepackage{bookmark} % fix pdf bookmark generation

\title{Patterns of crossing over and gene conversion in meiotic recombination}

\author{Christopher L Campbell}
%Albert Einstein College of Medicine}

\makeatletter\let\Title\@title\makeatother
\makeatletter\let\Author\@author\makeatother

% \newenvironment{abstract}%
% {\cleardoublepage\null \vfill\begin{center}%
% \bfseries \abstractname \end{center}}%
%     {\vfill\null}

\usepackage{tocbibind} % include toc within toc
\usepackage{titlesec}

\newcommand*\captionTitle[2]{\caption[#1]{#1#2}} % figure caption includes short title

\newenvironment{titemize} {
	\begin{itemize}
	  \setlength{\itemsep}{0cm}%
	  \setlength{\parskip}{0cm}%
} {\end{itemize}}

\newenvironment{tenumerate} {
	\begin{enumerate}
	  \setlength{\itemsep}{0cm}%
	  \setlength{\parskip}{0cm}%
} {\end{enumerate}}

\newcommand{\beginsupplement}{ % enable supplement numbering within chapters
    \setcounter{table}{0}
    \renewcommand{\thetable}{\thechapter.S\arabic{table}}%
    \setcounter{figure}{0}
    \renewcommand{\thefigure}{\thechapter.S\arabic{figure}}%
}
\newcommand{\beginmain}{ % reset supplement numbering within chapters
    \setcounter{table}{0}
    \renewcommand{\thetable}{\thechapter.\arabic{table}}%
    \setcounter{figure}{0}
    \renewcommand{\thefigure}{\thechapter.\arabic{figure}}%
}
%%% adjust spacing between figure number and label in listoffigures:
\makeatletter \renewcommand*\l@figure{\@dottedtocline{1}{1em}{3.2em}} \makeatother

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% \setcounter{tocdepth}{5}
% \setcounter{page}{1}

\frontmatter

%\begin{titlepage}
\begin{titlingpage}
\begin{center}
    \vspace*{1cm}
    \Large{ \textbf{\Title} } \\
    \vspace{1.5cm}
    \normalsize
    \Author \\ \vspace{2cm}
    \begin{tabularx}{\textwidth}{@{}X@{}X@{}}
        \begin{minipage}[t]{\linewidth}
            \textbf{Candidate:} \\ 
            %\vspace{1.5cm} \hrule\smallskip Signature
            \vspace{1.5cm} \hrule width 0.9\textwidth \smallskip Signature
            %\vspace{1.5cm} \noindent\rule{0.9\textwidth}{0.4pt}
        \end{minipage}
        & \\
        \begin{minipage}[t]{\linewidth}
            \vspace{1cm}
            \textbf{Thesis Advisor:} \\ 
            \vspace{1.5cm} \hrule width 0.9\textwidth \smallskip Signature
            \begin{flushleft}
                Adam Auton, D.Phil. \\
                \small \smallskip
                Assistant Professor, Department of Genetics \\
                Assistant Professor, Department of Epidemiology \& Population Health \\
            \end{flushleft}
        \end{minipage}
        \vspace{1cm}
        &
        \begin{minipage}[t]{\linewidth}
            \vspace{1cm}
            \textbf{Co-advisor:} \\ 
            \vspace{1.5cm} \hrule width 0.9\textwidth \smallskip Signature
            \begin{flushleft}
                Bernice E. Morrow, Ph.D. \\
                \small \smallskip
                Professor, Department of Genetics \\
                Professor, Department of Obstetrics \& Gynecology and Women's Health \\
                Professor, Department of Pediatrics (Cardiology) \\
                Sidney L. and Miriam K. Olson Chair in Cardiology \\
                Director, Division of Translational Genetics, Department of Genetics \\
            \end{flushleft}
        \end{minipage}\\
    \end{tabularx}
%%%
\vfill
Submitted in partial fulfillment of the requirements for the Degree of Doctor of
Philosophy in the Graduate Division of Medical Sciences.
\vspace{0.5cm}
\\ Albert Einstein College of Medicine
\\ Yeshiva University
\\ New York
\\ May 31, 2016
\end{center}
%\end{titlepage}
\end{titlingpage}
%\newpage
\setcounter{page}{1}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\chapter{Abstract}
\addcontentsline{toc}{chapter}{Abstract}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Abstract: The abstract of the Dissertation is to include: a hypothesis, the procedures followed, the
% significant results and the general conclusions. The abstract is to be presented on a separate page
% headed with the word ABSTRACT in capital letters centered on the page. On the next line is the
% title of the Dissertation. The following line is the full name of the student. The length of the
% abstract must not exceed 600 words. (Please note the separate instructions for the 350 word
% microfilm copy abstract described in the first section of this manual.)
\thispagestyle{plain}
\begin{centering} ABSTRACT \\ \vspace{10pt} %\end{centering}
\Title \\
\Author \\
\end{centering}
\vspace{10pt}
 \DoubleSpacing


Recombination during meiosis is an essential process to the generation of gametes.
It acts to shuffle genetic variation within the genome, generating new combination of alleles that can be subject to natural selection.
Failures of the chromosomes to synapse and recombine can result in aneuploidy, while improper repair of double strand breaks can cause chromosome rearrangements, both of which are highly deleterious.

Recombination has been studied with a wide variety of approaches, including molecular methods such as sperm typing, and inferential methods, such as linkage disequilibrium modeling and pedigree analysis.
Most research to date has focused on crossover, the reciprocal exchange of genetic material between homologous chromosomes.
Crossovers vary widely in frequency and placement within the genome as well as between individuals, sexes, and populations, and different species.
Multiple factors affect the placement of crossovers within the genome, including the PRDM9 protein, which has been shown to localize crossovers to concentrated areas of recombination known as hotspots.
In addition, crossover interference acts to inhibit crossovers in close proximity.
It has also been suggested that recombination properites could change with age.

In humans, I present a pedigree analysis with data on over 18,000 meioses in males and females.
Using this data, I further characterize sex differences in human recombination, and create a sex-specific map of recombination in the human genome.
This results reinforce previous work showing that females have a higher recombination rate than males, by a ratio of 1.6 in the autosomes, and that male recombination tends to be focused towards the telomeres.
In addition, I find that males and females have a difference in the proportion of crossovers that overlap known hotspots, with males having 4.6\% higher hotspot overlap.
Finally, I measure crossover interference, which affects the spatial positioning of crossovers.
I find that in older mothers there is a steep increase in crossovers that escape regulation by interference, and appear closely spaced, pointing to a possible deregulation of recombination that increases with age in females.

Additionally, I analyze crossovers in an inbred domestic dog pedigree, consisting of 408 meioses.
Dogs are unique in that PRDM9 in this species has accumulated mutations, rendering it inactive, and raising question as to how crossovers are placed without this regulatory mechanism.
Using this data to analyze sex differences in dogs, I find that females have a higher recombination rate than males, a ratio of 1.2 within the autosomes, and that male recombination in dogs strongly prefers the telomeric regions.
I find that, despite the absence of PRDM9, dog recombination appears to be more concentrated to a smaller proportion of sequence when compared to humans.
Finally, I show evidence for positive crossover interference acting in the dog genome with a similar mechanism to what has been seen in humans.

An alternate outcome of recombination is non-crossover, also known as gene conversion, the uni-directional transfer of small amounts of genetic material that occurs during the repair of double strand breaks in meiosis.
Gene conversion can result in non-Mendelian transmission by converting an allele on the homologous receptor chromosome to that of the donor without reciprocal exchange, and plays an important role in shaping patterns of linkage disequilibrium throughout the genome.
Here, I develop a hidden Markov model to detect gene conversion events in admixed population genetic data.
The model uses two divergent ancestral reference populations in order to predict the location of gene conversion events on an admixed haplotype.
Simulation results show that the model can plausibly be used to detect gene conversion, however the complexity is high, with a large number of states at each site.

Overall, I have provided further evidence for sexual dimorphism in crossover placement and frequency within the human and dog genomes. Additionally, I have demonstrated the effects of the loss of PRDM9 on the recombination landscape in dogs, and through comparison to humans, gained additional information on recombination in both species.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Acknowledgements}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% The Training Program in Cellular and Molecular Biology and Genetics, T32 GM007491.

\SingleSpacing
\clearpage
\tableofcontents
\listoffigures
\listoftables

\glsaddall
\clearpage\begingroup\let\newpage\relax
    \printglossary[type=main,style=long,nonumberlist,title={List of Abbreviations}]
    \addcontentsline{toc}{chapter}{List of Abbreviations}
\endgroup
\include{abbrv}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\mainmatter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\chapterstyle{ger}
 \DoubleSpacing
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Introduction} \label{ch:introduction}
\include{introduction/introduction}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{SingleSpace}
\chapter{Escape from crossover interference increases with maternal age} \label{ch:cointEsc}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\noindent Christopher L. Campbell$^1$, Nicholas A. Furlotte$^2$, Nick Eriksson$^{2,\dagger}$, David Hinds$^2$, and Adam Auton$^1$

\vspace{0.5cm}
\noindent This manuscript has been published: \\
Campbell, C. L. et al. Escape from crossover interference increases with maternal age. Nat. Commun. 6:6260 doi: 10.1038/ncomms7260 (2015). \\
PMCID: PMC4335350

\vspace{0.5cm}
\noindent $^1$ Department of Genetics, Albert Einstein College of Medicine, 1301 Morris Park Avenue, Bronx, New York 10461, USA. \\
\noindent $^2$ 23andMe Inc., Mountain View, California 94043, USA. \\
\noindent $^\dagger$ Present address: Coursera, 381 East Evelyn Avenue, Mountain View, California 94041, USA. \\

\vspace{0.5cm}
\begin{centering}
    Correspondence and requests for materials should be addressed to \\
    A.A.\ (email: \texttt{adam.auton@einstein.yu.edu}). \\
\end{centering}
\end{SingleSpace}

\include{cointEscape/cointEscape}
\include{cointEscape/cointEscapeSupp}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{SingleSpace}
\chapter{Crossover interference varies by age and individual} \label{ch:cointExtras}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\noindent Christopher L. Campbell$^1$ and Adam Auton$^{1*}$

\vspace{0.5cm}
\noindent This chapter contains unpublished data.\\

\vspace{0.5cm}
\noindent $^1$ Department of Genetics, Albert Einstein College of Medicine, 1301 Morris Park Avenue, Bronx, New York 10461, USA. \\
\noindent $^*$ Former affiliation.
\end{SingleSpace}

\include{cointExtras/cointExtras}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{SingleSpace}
\chapter{A pedigree-based map of recombination in the domestic dog genome} \label{ch:dogPed}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\noindent Christopher L Campbell$^1$ , Claude Bh\'{e}rer$^{1,2}$ , Bernice E Morrow$^1$ , Adam Boyko$^3$ , and Adam Auton$^{1*}$

\vspace{0.5cm}
\noindent This manuscript has been submitted to \textit{Genetics}. \\

\vspace{0.5cm}
\noindent $^1$ Department of Genetics, Albert Einstein College of Medicine, 1301 Morris Park Avenue, Bronx, New York 10461, USA. \\
\noindent $^2$ New York Genome Center, New York, New York 10013, USA \\
\noindent $^3$ Department of Biomedical Sciences, College of Veterinary Medicine, Cornell University, Ithaca, New York 14853, USA \\
\noindent $^*$ Former affiliation.

\vspace{0.5cm}
\begin{centering}
    Correspondence and requests for materials should be addressed to \\
    A.A.\ (email: \texttt{adam.auton@gmail.com}). \\
\end{centering}
\end{SingleSpace}

\include{dogPed/dogPed}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{SingleSpace}
\chapter{Detection of gene conversion in human admixed population genetic data} \label{ch:geneConv}
%\chapter[Detection of gene conversion]{Detection of gene conversion in admixed population genetic data}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\noindent Christopher L. Campbell$^1$ and Adam Auton$^{1*}$

\vspace{0.5cm}
\noindent This chapter contains unpublished data.\\

\vspace{0.5cm}
\noindent $^1$ Department of Genetics, Albert Einstein College of Medicine, 1301 Morris Park Avenue, Bronx, New York 10461, USA. \\
\noindent $^*$ Former affiliation.
\end{SingleSpace}

\include{geneConv/geneConv}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Discussion} \label{ch:discussion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\include{discussion/discussion}

\backmatter

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Appendix}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\includepdf[pages=1-7]{appendix/Campbell2015.pdf}
%\includepdf[pages=1-28]{appendix/Campbell2015_supp.pdf}
%\includepdf[pages=1-20]{appendix/SexDimorphismRecombination_2016April8_maintext.pdf}
%\includepdf[pages=1-21]{appendix/SexDimorphismRecombination_2016April8_SUPP.pdf}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
